src/qt/test/compattests.cpp

```diff
@@ -2,7 +2,13 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include "config/bitcoin-config.h"
+#endif
+
+#if defined(ENABLE_WALLET) && defined(ENABLE_BIP70)
 #include "paymentrequestplus.h" // this includes protobuf's port.h which defines its own bswap macos
+#endif
```

Comment:
Do we need to get bswap macros from somewhere else?

Reply:
No, we have our own bswap macros. The only reason this is tested is that there was a collision between our bswap macros and protobuf's. So commenting this out is harmless.